terraform/cp4i/outputs.tf

```diff
@@ -16,6 +16,9 @@ output "cluster_name" {
 output "kubeconfig" {
   value = data.ibm_container_cluster_config.cluster_config.config_file_path
 }
+output "flavors" {
```

Comment:
Not sure we need this exposed as output since it is a parameter that the user has entered, so they would know this.